docs/packages/library/README.md

```diff
@@ -61,6 +61,39 @@ project. This includes JavaScript files that would be available to your compiled
 
 ## Quickstart
 
+The fasted way to get started is by using the `create-project` scaffolding tool.
+Don’t want to use the CLI helper? No worries, we got you covered with the [manual installation](#manual-installation).
```

Comment:
Either "No worries; we have got you covered" or "No worries; we have you covered"